### DISCUSSION TITLE
Use Transformer for one Group in React

### DISCUSSION THREAD

**Discussion by ddrepkk89** [konvajs/konva#1334] [Q&A]
Dear Gurus in community, I'm using Konva within React. I want create a group with 2 circles. And then apply Transformer on this group to let me  drag and zoom the group. I have reviewed the example on the homepage, but I can't make it work in React. Can anyone help?

Thanks in advance.

**Comment by lavrton** [comment]
What did you try? What exactly doesn't work for you?
